next.config.js

```diff
@@ -5,6 +5,11 @@ const nextConfig = {
   reactStrictMode: true,
   async redirects() {
     return [
+      {
+        source: '/',
+        destination: '/docs',
+        permanent: true,
+      },
```

Comment:
Ensures root path redirects to `/docs`. This allows contributors to more easily see their changes on Vercel without having to remember to append the link with `/docs`.